main/src/com/google/refine/commands/row/GetRowsCommand.java

```diff
@@ -149,27 +205,19 @@ protected void internalRespond(HttpServletRequest request, HttpServletResponse r
             }
 
             Engine engine = getEngine(request, project);
-            String callback = request.getParameter("callback");
```

Comment:
I missed this during my review, but the removal of JSONP support will break existing callers (like the GData extension which doesn't currently work). Instead of being silently ignored, the server should return a clear error if JSONP is not being supported.